components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -525,6 +535,13 @@ protected boolean isAccountLock(String userName, UserStoreManager userStoreManag
             throw new AccountLockException("Error occurred while retrieving " + AccountConstants
                     .ACCOUNT_LOCKED_CLAIM + " claim value", e);
         }
+        if (isAccountLockBypassRoleExist(userStoreManager, userName)) {
+            String error = String.format("Account lock feature is bypassed as lock bypass role is " +
+                    "attached on to the user %s", userName);
+            log.info(error);
```

Comment:
This should be ab error log. Not info log

Reply:
The purpose of this audit log is to let know that user is a privilege user hence bypass account locking feature. Is this not required? (Since this is a special case, this IMO is not an error)